identity_stardust/examples/ex3_deactivate_did.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity_stardust::StardustClientExt;
+use identity_stardust::StardustDocument;
+use iota_client::block::address::Address;
+use iota_client::secret::SecretManager;
+use iota_client::Client;
+
+mod ex0_create_did;
+
+/// Demonstrate how to deactivate a DID in an Alias Output.
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+  // Create a new DID in an Alias Output for us to modify.
+  let (client, _, secret_manager, document): (Client, Address, SecretManager, StardustDocument) =
+    ex0_create_did::run().await?;
+
+  // Deactivate the DID by publishing an empty document.
+  // This process can be reversed since the Alias Output is not destroyed.
```

Comment:
The example should also demonstrate how to re-publish the DID Document, "reactivating" the DID.